### ISSUE TITLE
Cant find the transcribed note. Is their a default how it will be labeld after transcription? 

### ISSUE THREAD

**Issue by aaronmaas** [drewmcdonald/obsidian-magic-mic#3]
Recording works fine. Audio file in attachments. Running command "transcribe and summarize". the assistant is the default one. open AI key generated and included in settings.

**Comment by drewmcdonald** [comment]
Hey @aaronmaas, it'll be titled `Magic Mic - ${date.format('yyMMDDHHmmss')}`.

Let me know if you're not able to find it. I have a suspicion about what could be going wrong.

**Comment by aaronmaas** [comment]
Hey there @drewmcdonald thanks for your answer. I searched again for the specific notes as you outlined above, but they do not exist :) so whats your suspicion?

**Comment by drewmcdonald** [comment]
Can you share a little more about the audio files you've seen created? I'm wondering if we're allowing a file type that OpenAI does not.

**Comment by drewmcdonald** [comment]
And is there anything exceptional about the audio files or your settings?

**Comment by drewmcdonald** [comment]
I think some filetypes were not playing nice with a little optimization I had to skip processing on smaller files. Version 1.0.5 might fix it! Give it a shot.

**Comment by vikram11** [comment]
This is almost perfect and what I have been searching for. Just one thing: can it just paste the transcription (or whatever my assistant does with it) at the cursor or just stick it in the clipboard? I like to dictate into notes. I don't want to create a new note for every piece and have to keep copy/pasting into the note that I'm writing. Fantastic work. Thank you. If this one thing is fixed I'll be using it daily.

**Comment by aaronmaas** [comment]
> Can you share a little more about the audio files you've seen created? I'm wondering if we're allowing a file type that OpenAI does not.

Yes of cource. It is a .webm file type. Your updated version startet to do process the transcibing but still there is no outcome. Is there any file type you would recommend? I think the .webm is the default when I use magic mic or the obsidian audio recorder. Thanks for the effort :) 

The audiofiles are labled: magic_mic_20241009151213.webm 
Other exceptional settings I cannot identify.

**Comment by jayprob** [comment]
Hello, 
I'm having the exact same issue as aaronmaas. When I try to transcribe, I get the message: Magic Mic: transcribing (which stays on screen until I close Obsidian), but no transcription is created. Thanks for any help you can give me.
